examples/cmake/project/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+cmake_minimum_required(VERSION 3.14.0 FATAL_ERROR)
```

Comment:
3.14 is required for `FetchContent`, but the stuff here should allow using an older version of CMake.